ecc/bls12381/gt.go

```diff
@@ -8,13 +8,14 @@ const GtSize = ff.Fp12Size
 // Gt represents an element of the output group of a pairing.
 type Gt ff.Fp12
 
-func (z Gt) String() string        { return (ff.Fp12)(z).String() }
-func (z *Gt) Set(x *Gt)            { (*ff.Fp12)(z).Set((*ff.Fp12)(x)) }
-func (z *Gt) SetBytes(b []byte)    { (*ff.Fp12)(z).SetBytes(b) }
-func (z *Gt) Bytes() []byte        { return (*ff.Fp12)(z).Bytes() }
-func (z *Gt) SetOne()              { (*ff.Fp12)(z).SetOne() }
-func (z *Gt) IsEqual(x *Gt) bool   { return (*ff.Fp12)(z).IsEqual((*ff.Fp12)(x)) }
-func (z *Gt) Exp(x *Gt, n *Scalar) { (*ff.Fp12)(z).Exp((*ff.Fp12)(x), n.Bytes()) }
-func (z *Gt) Mul(x, y *Gt)         { (*ff.Fp12)(z).Mul((*ff.Fp12)(x), (*ff.Fp12)(y)) }
-func (z *Gt) Sqr(x *Gt)            { (*ff.Fp12)(z).Sqr((*ff.Fp12)(x)) }
-func (z *Gt) Inv(x *Gt)            { (*ff.Fp12)(z).Inv((*ff.Fp12)(x)) }
+func (z Gt) String() string           { return (ff.Fp12)(z).String() }
```

Comment:
Gt isn't using cyclo6, which seems odd to me.

Reply:
take a look on the new approach.